trieste/acquisition/rule.py

```diff
@@ -117,6 +117,50 @@ def acquire(
         return point, None
 
 
+class MaxValueEntropySearch(AcquisitionRule[None, SearchSpace]):
```

Comment:
do we have a need for a new rule? it seems to me that MES should be an acquisition function only, that uses EfficientGlobalOptimization

Reply:
The issue is that MES requires some extra arguments that govern the performance of the approximations its uses. A single EfficientGlobalOptimization rule would make more sense, but will need to be able to take generic extra arguments when initializing.

Reply:
it appears the need to add arguments is arising from requiring the rule to have MES as a default argument. I would say the correct flow would be to pass `MES` to `EfficientGlobalOptimization`:
```python
trieste.acquisition.rule.EfficientGlobalOptimization(trieste.acquisition.MES(search_space, num_samples=10, grid_size=5000))
```
Does that make sense?

Reply:
I had missed that this was what the EfficientGlobalOptimization class was for! I've got rid of the extra MES rule now.

Reply:
great. That you'd missed it suggests we need to document it more loudly or clearly